types/node/index.d.ts

```diff
@@ -5831,3 +5832,818 @@ declare module "async_hooks" {
      */
     export function createHook(options: HookCallbacks): AsyncHook;
 }
+
+declare module "http2" {
+    import * as events from "events";
+    import * as fs from "fs";
+    import * as http from "http";
+    import * as net from "net";
+    import * as stream from "stream";
+    import * as tls from "tls";
+    import * as url from "url";
+
+    export interface IncomingHttpHeaders extends http.IncomingHttpHeaders {}
```

Comment:
This could be:
```ts
import { IncomingHttpHeaders, OutgoingHttpHeaders } from "http";
export { IncomingHttpHeaders, OutgoingHttpHeaders } from "http";
```